testsuite/cluster-test/src/atomic_histogram.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+#![forbid(unsafe_code)]
+
+use crate::tx_emitter::TxStats;
+use std::sync::atomic::{AtomicU64, Ordering};
+
+const HISTOGRAM_CAPACITY: usize = 2048;
```

Comment:
We can probably just have both of capacity and width(maybe 'step' is better?) configurable?
Have them as fields on AtomicHistogram struct. Default can read those constants, but for those who want customize it, they could use `new(capacity, size)`.